app/flags.go

```diff
@@ -57,4 +64,25 @@ var (
 		Usage:   "The resource-version (etag) to update from, if not set the cli will use the latest (optional)",
 		Aliases: []string{"v"},
 	}
+	APIKeyFlag = &cli.StringFlag{
+		Name:     APIKeyFlagName,
+		Usage:    "The API Key used for authentication (" + APIKeyVersionTag + ")",
```

Comment:
I wonder, with feature flagging we could remove the APIKeyVersionTag after this merge.